smoke/README.md

```diff
@@ -0,0 +1,105 @@
+# QA for OpenShift Shared Resource CSI Driver
+This is a Behavior-driven development (or BDD) test framework leveraging `behave` for `python`. BDD is an agile software development technique that encourages collaboration between developers, QA and non-technical or business participants in a software project.Behave uses tests written in a natural language style, backed up by Python code.
+
+We are using this framework to achieve the QA verification of any Jira Stories development takes on for the OpenShift Shared Resource CSI Driver, as well as any regression test suites the team wants to cultivate for the OpenShift Shared Resource CSI Driver.
+
+Hence, the reader should consider these tests the 'next phase' of testing after the [golang based tests](https://github.com/openshift/csi-driver-shared-resource/tree/master/test/e2e) that run in OpenShift CI prow jobs as part of merging pull requests with code changes.
+
+
+# Run smoke test on local machine
+<pre>
+- oc login [cluster-server] -u username -p password --kubeconfig=kubeconfig
+- export KUBECONFIG=kubeconfig
+- make smoke
+</pre>
```

Comment:
Use triple-backticks for GitHub Markdown - ignore my slash since it breaks the suggestions
```suggestion
\```sh
$ oc login [cluster-server] -u username -p password --kubeconfig=kubeconfig
$ export KUBECONFIG=kubeconfig
$ make smoke
\```
```